test/sqllogictest/introspection/attribution_sources.slt

```diff
@@ -31,58 +31,52 @@ CREATE INDEX v_idx_x ON v(x);
 statement ok
 SELECT mz_unsafe.mz_sleep(8)
 
-query IT
-SELECT id, global_id FROM mz_internal.mz_dataflow_global_ids ORDER BY id, global_id;
-----
-4  u2
-4  u3
-
 query TI
-SELECT global_id, lir_id FROM mz_internal.mz_lir_mapping ORDER BY global_id, lir_id DESC;
+SELECT export_id, lir_id FROM mz_internal.mz_lir_mapping ORDER BY export_id, lir_id DESC;
 ----
-u2  5
-u2  4
-u2  3
-u2  2
-u2  1
 u3  7
 u3  6
+u3  5
+u3  4
+u3  3
+u3  2
+u3  1
 
 ## attribution queries
 
 # omitting sum(duration_ns) as duration, sum(count) as count
 query TIIT
-SELECT global_id, lir_id, parent_lir_id, REPEAT(' ', nesting * 2) || operator AS operator
+SELECT export_id, lir_id, parent_lir_id, REPEAT(' ', nesting * 2) || operator AS operator
     FROM           mz_internal.mz_lir_mapping mlm
          LEFT JOIN mz_introspection.mz_compute_operator_durations_histogram mcodh
          ON (mlm.operator_id_start <= mcodh.id AND mcodh.id < mlm.operator_id_end)
-GROUP BY global_id, lir_id, operator, parent_lir_id, nesting
-ORDER BY global_id, lir_id DESC;
+GROUP BY export_id, lir_id, operator, parent_lir_id, nesting
+ORDER BY export_id, lir_id DESC;
 ----
-u2  5  NULL  Join::Differential␠2␠»␠4
-u2  4  5  ␠␠Arrange␠3
-u2  3  4  ␠␠␠␠Get::Collection␠u1
-u2  2  5  ␠␠Arrange␠1
-u2  1  2  ␠␠␠␠Get::Collection␠u1
 u3  7  NULL  Arrange␠6
 u3  6  7  ␠␠Get::PassArrangements␠u2
+u3  5  NULL  Join::Differential␠2␠»␠4
+u3  4  5  ␠␠Arrange␠3
+u3  3  4  ␠␠␠␠Get::Collection␠u1
+u3  2  5  ␠␠Arrange␠1
+u3  1  2  ␠␠␠␠Get::Collection␠u1
 
 # omitting pg_size_pretty(sum(size)) as size
 query TIIT
-SELECT global_id, lir_id, parent_lir_id, repeat(' ', nesting * 2) || operator AS operator
+SELECT export_id, lir_id, parent_lir_id, repeat(' ', nesting * 2) || operator AS operator
     FROM           mz_internal.mz_lir_mapping mlm
          LEFT JOIN mz_introspection.mz_arrangement_sizes mas
          ON (mlm.operator_id_start <= mas.operator_id AND mas.operator_id < mlm.operator_id_end)
-GROUP BY global_id, lir_id, operator, parent_lir_id, nesting
-ORDER BY global_id, lir_id DESC;
+GROUP BY export_id, lir_id, operator, parent_lir_id, nesting
+ORDER BY export_id, lir_id DESC;
 ----
-u2  5  NULL  Join::Differential␠2␠»␠4
-u2  4  5  ␠␠Arrange␠3
-u2  3  4  ␠␠␠␠Get::Collection␠u1
-u2  2  5  ␠␠Arrange␠1
-u2  1  2  ␠␠␠␠Get::Collection␠u1
 u3  7  NULL  Arrange␠6
 u3  6  7  ␠␠Get::PassArrangements␠u2
```

Comment:
This might be a blocker for merging this: The attribution query output now references a dataflow `u2` that we don't show anywhere, which is confusing. We probably need to do https://github.com/MaterializeInc/database-issues/issues/8845 first.